registry.json

```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+  {
+    "ticker": "MONA",
+    "poolId": "6a84654c217f1923b086e076192b50042911c7b7af3987a1992b77e9",
```

Comment:
I checked your pool and it says your pool's pledge has not been met. I'm going to request you fix this before I approve if only to raise your awareness of the situation...

Reply:
Had to wait for the end of an epoch